.gitmodules

```diff
@@ -16,7 +16,7 @@
 [submodule "mongodb_exporter"]
 	path = sources/mongodb_exporter/src/github.com/percona/mongodb_exporter
 	url = https://github.com/percona/mongodb_exporter.git
-	branch = release-0.36.0
+	branch = release-0.37.0
```

Comment:
Why we are not using `main` here as we do for other exporters?

Reply:
we actually should use release branches for all exporters :)